### PR TITLE
add logger for ConfigurationService

### PR DESCRIPTION
**Pull request by alicer3**
add logger
modify some comment

### CODE REVIEW

File: bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt [hunk 1]
```diff
@@ -73,6 +72,9 @@ fun List<SecureHash>.hash(): SecureHash = map {
  */
 @CordaService
 class ConfigurationService(appServiceHub : AppServiceHub): SingletonSerializeAsToken()  {
```
Comment (mplattr3) [inline bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt]: `appServiceHub` is unused.

File: bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt [hunk 2]
```diff
@@ -82,10 +84,23 @@ class ConfigurationService(appServiceHub : AppServiceHub): SingletonSerializeAsT
         else {
             val configResource = this::class.java.classLoader.getResource(fileName)
             if (configResource == null) {
-                //logger.error("Configuration $configName.conf has not been found")
                 null
             } else ConfigFactory.parseFile(File(configResource.toURI()))
```
Comment (mplattr3) [inline bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt]: `parseFile()` can throw `ConfigException`

File: bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt (hunk 1 above)
Comment (alicer3) [reply bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt]: it is actually required by SingletonSerializeAsToken. I actually tried to remove it before, and it caused error.

Comment (alicer3) [reply bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt]: optimize import will do

File: bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt (hunk 2 above)
Comment (alicer3) [reply bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt]: will do

**Comment by alicer3** [comment]
@mplattr3 I think I have done the change you requested.

**Comment by alicer3** [comment]
@ivanr3 
Hi Ivan, seems like Moritz is on holiday. Can you help me merge this PR?

File: bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt [hunk 3]
```diff
@@ -1,58 +1,52 @@
...
-import net.corda.core.serialization.deserialize
-import net.corda.core.transactions.SignedTransaction
+import net.corda.core.utilities.loggerFor
 import sun.security.util.ByteArrayLexOrder
 import java.io.File
-import java.lang.StringBuilder
 import java.nio.file.Paths
-import java.sql.Blob
-import java.util.*
+import javax.naming.ConfigurationException
```
Comment (mplattr3) [inline bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt]: Unused import

**Pull request merged**